integrationTest/integration_test.go

```diff
@@ -970,15 +970,16 @@ func makeTests(t *testing.T) []*TestGroup {
 			// of record. When the provider fixes the bug or changes behavior,
 			// update the AuditRecords().
 
-			//clear(),
-			//tc("a 255-byte TXT", txt("foo255", strings.Repeat("C", 255))),
-			//clear(),
-			//tc("a 256-byte TXT", txt("foo256", strings.Repeat("D", 256))),
-			//clear(),
-			//tc("a 512-byte TXT", txt("foo512", strings.Repeat("C", 512))),
-			//clear(),
-			//tc("a 513-byte TXT", txt("foo513", strings.Repeat("D", 513))),
-			//clear(),
+			tc("a 254-byte TXT", txt("foo254", strings.Repeat("B", 254))),
```

Comment:
Its good that these work for Hetzner, but let's not re-enable the tests for all providers.  My plan is to rework how txt records are handled in a few months and revise all the tests at that time.